.well-known/ratify-verification.crt

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Don't think it's required here. But I thought it might be ok to keep in both places? Potentially two different trusted sources for certificate distribution? Do you have a strong preference?

Reply:
Gotcha, I don't have a strong opinion on it, we can keep it for now.